packages/h5p-server/src/H5PEditor.ts

```diff
@@ -126,6 +130,9 @@ export default class H5PEditor {
             temporaryStorage,
             this.config
         );
+        this.contentUserDataManager = new ContentUserDataManager(
+            contentUserDataStorage
```

Comment:
What happens when contentUserDataStorage is undefined?

Reply:
The contentUserDataManger is constructed, but if the methods are called they return `undefined`:

https://github.com/Lumieducation/H5P-Nodejs-library/pull/1886/files#diff-175df6189e4a5c5418312fab8fcb2634717e3565726190172a885f90a9fd26e7R73

Should I use another way of doing this?

Reply:
I see. I'd have implemented this in H5PEditor, but it can also be done in ContentUserDataManager